### PR TITLE
Math nodes

### PR DESCRIPTION
**Pull request by isiko**
Adds nodes for getting the Maximum and Minimum of Values, checking if values are equal, and a node for logging values to the console.

### CODE REVIEW

**Comment by isiko** [comment]
Whoops, didn't see your push @Keavon, I assume that was just a rebase?

File: node-graph/gcore/src/ops.rs [hunk 1]
```diff
@@ -83,6 +83,42 @@ where
...
+	}
+}
+
+// Equality
+pub struct EqParameterNode<Second> {
+	second: Second,
+}
+
+#[node_macro::node_fn(EqParameterNode)]
+fn min<T: core::cmp::PartialOrd>(first: T, second: T) -> bool {
```
[the remark this answers is not in the document]
Comment (isiko) [reply node-graph/gcore/src/ops.rs]: Fixed :+1:

File: editor/src/messages/portfolio/document/node_graph/node_graph_message_handler/node_properties.rs [hunk 2]
```diff
@@ -1022,6 +1022,33 @@ pub fn exponent_properties(document_node: &DocumentNode, node_id: NodeId, _conte
...
+
+pub fn eq_properties(document_node: &DocumentNode, node_id: NodeId, _context: &mut NodePropertiesContext) -> Vec<LayoutGroup> {
+	let operand = |name: &str, index| {
+		let widgets = number_widget(document_node, node_id, index, name, NumberInput::default(), true);
+
+		LayoutGroup::Row { widgets }
+	};
+	vec![operand("Equality", 1)]
+}
+
```
Comment (TrueDoctor) [inline editor/src/messages/portfolio/document/node_graph/node_graph_message_handler/node_properties.rs:1051]: Can you compactify all of these by creating a higher-order factory function?
(and the other math properties)

Comment (TrueDoctor) [reply editor/src/messages/portfolio/document/node_graph/node_graph_message_handler/node_properties.rs:1051]: I think we can just make this a separate pr, @Ezbaze would you be interested in working on that?

**Review by TrueDoctor — APPROVED** [review]

**Pull request merged**